docs/api-reference/v3-sdk.md

```diff
@@ -1,7 +1,10 @@
 import FeedbackComponent from "@site/src/pages/feedback.md";
 
-# SDK v5.x.x to v3.x.x
+# SDK 3.x.x to v5.x.x
 
-The SDK v5.x.x to v3.x.x API docs can be viewed [here](https://lit-js-sdk-v3-api-docs.vercel.app/).
+The SDK v3.x.x to v5.x.x API docs can be viewed [here](https://lit-js-sdk-v3-api-docs.vercel.app/). 
 
+:::warning
+Please note that the latest API docs are v6.x.x.
```

Comment:
Should we just link to https://developer.litprotocol.com/api-reference/latest, so we don't have to update the major version number each time?

Reply:
I have now changed the `v6.x.x` to not be a specific version number, and instead be the latest link